modules/backup-etcd.adoc

```diff
@@ -14,26 +14,37 @@ Only save a backup from a single master host. Do not take a backup from each mas
 
 .Prerequisites
 
-* SSH access to a master host.
-
-.Procedure
-
-. Access a master host.
-
-. If the cluster-wide proxy is enabled, be sure that you have exported the `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have checked whether the cluster-wide proxy is enabled.
```

Comment:
Do you need the _You have_ wording for these?
* access to the cluster as a user with the `cluster-admin` role
* cluster-wide proxy is enabled

Reply:
Yeah the proxy doesn't have to be enabled as a prereq, we're just checking if it is nor not. I'm not sure if I could convey that properly without starting with the "You have"